### ISSUE TITLE
NPM / Yarn FileUpdater LoadError

### ISSUE THREAD

**Issue by jimcavoli** [dependabot/dependabot-core#815]
I'm getting a strange error when trying to require in 'dependabot/file_updaters' or 'dependabot/update_checkers' which ultimately are throwing the same `LoadError`:
```
`require': cannot load such file -- dependabot/file_updaters/java_script/npm_and_yarn//npmrc_builder (LoadError)
```

Bizarrely, this only happens on the first attempt to require, but the second works:
```
[1] pry(main)> require 'dependabot/file_updaters'
LoadError: cannot load such file -- dependabot/file_updaters/java_script/npm_and_yarn//npmrc_builder
from [...]/kernel_require.rb:32:in `require'
[2] pry(main)> require 'dependabot/file_updaters'
=> true
```

I'm pretty sure the culprit is the way https://github.com/dependabot/dependabot-core/blob/master/lib/dependabot/update_checkers/java_script/npm_and_yarn/version_resolver.rb#L11-L13 are put together, specifically the trailing `/` at https://github.com/dependabot/dependabot-core/blob/master/lib/dependabot/update_checkers/java_script/npm_and_yarn/version_resolver.rb#L11

I can work on a PR for this if need be, but I'm confused as to how it isn't already breaking the specs in the first place.

Additional info:
* Ruby 2.5.3
* ASDF Version Manager
* Rubygems 2.7.6
* Dependabot Core 0.75.110

**Comment by greysteil** [comment]
Sorry about that - fixed in https://github.com/dependabot/dependabot-core/commit/2fac1ad10886553f40bf5f46a73d8b89fb89fb3d, which I'll release today.

**Comment by jimcavoli** [comment]
Hey, no problem; thanks on the super fast fix 👍 👍

**Comment by greysteil** [comment]
Released in `v0.75.111`. Thanks for the heads up on this one - weirdly it looks like it was only affecting setups that required the `file_updaters` first.
